### ISSUE TITLE
Request for training codes

### ISSUE THREAD

**Issue by jiunyen-ching** [Adelaide-AI-Group/PALNet#1]
Hi, thanks for the amazing work! 

I would like to study your codes more especially on implementing the loss function and later try training on my own data. However I noticed that the available codes are only for testing.

May I request for the training codes to be published too? Thank you in advance!
